### ISSUE TITLE
cannot connect to X server 

### ISSUE THREAD

**Issue by DenisSergeevitch** [xingyizhou/CenterTrack#2]
Hi, I'm trying to test demo in Colab but getting error: "cannot connect to X server" as a result. 

I cant find what I should comment in code to get it working directly from the terminal, any advice will be appreciated.

**Here is full output:**
`/content/CenterTrack/src
/usr/local/lib/python3.6/dist-packages/sklearn/utils/linear_assignment_.py:22: FutureWarning: The linear_assignment_ module is deprecated in 0.21 and will be removed from 0.23. Use scipy.optimize.linear_sum_assignment instead.
  FutureWarning)
Running tracking
Using tracking threshold for out threshold! 0.1
Fix size testing.
training chunk_sizes: [32]
input h w: 448 800
heads {'hm': 10, 'reg': 2, 'wh': 2, 'tracking': 2, 'dep': 1, 'rot': 8, 'dim': 3, 'amodel_offset': 2}
weights {'hm': 1, 'reg': 1, 'wh': 0.1, 'tracking': 1, 'dep': 1, 'rot': 1, 'dim': 1, 'amodel_offset': 1}
head conv {'hm': [256], 'reg': [256], 'wh': [256], 'tracking': [256], 'dep': [256], 'rot': [256], 'dim': [256], 'amodel_offset': [256]}
Creating model...
Using node type: (<class 'model.networks.dla.DeformConv'>, <class 'model.networks.dla.DeformConv'>)
Warning: No ImageNet pretrain!!
loaded ../models/nuScenes_3Dtracking.pth, epoch 70
: cannot connect to X server `

**Comment by xingyizhou** [comment]
This is because we use cv2.imshow to visualize the predictions. You will need a screen for that. For Colab, you will need to use matplotlib to visualize images.

**Comment by DenisSergeevitch** [comment]
@xingyizhou is not it possible to write visualization directly to mp4 file at this moment?

**Comment by xingyizhou** [comment]
For `demo.py`, it can. You can add `--save_video --video_h xxx --video_w` to your command and comment out all `cv2.imshow` and `cv2.waitKey` in `demo.py`.

**Comment by bhack** [comment]
You could use unmodifed code with https://github.com/jupyter/notebook/issues/3935#issuecomment-493722525

**Comment by sisrfeng** [comment]
> For `demo.py`, it can. You can add `--save_video --video_h xxx --video_w` to your command and comment out all `cv2.imshow` and `cv2.waitKey` in `demo.py`.

1. Also need to replace "cv2.imshow" in function show_all_image in "debugger.py" with "pass"
2.  in demo.py:
 ```
       try:
           cv2.imshow('input', img)
         except:
           print('FINISH!')
           save_and_exit(opt, out, results, out_name)
```
should be:
```
       if img is not None:
          print('go')
        else:
          print('FINISH!')
          save_and_exit(opt, out, results, out_name)
```
